brat/test/golden/error/fanout-too-many-overs.brat.golden

```diff
@@ -0,0 +1,6 @@
+Error in test/golden/error/fanout-too-many-overs.brat@FC {start = Pos {line = 2, col = 5}, end = Pos {line = 2, col = 13}}:
+f = { [/\] }
+    ^^^^^^^^
+
+  Type error: No unders but overs: (head :: Nat) for [/\]
```

Comment:
The Fanout typechecking code just returns the fresh qubits, so this is happening when checking `a, b, c :: A, B`